Configuration/Eras/python/Era_Run3_cff.py

```diff
@@ -12,8 +12,9 @@
 from Configuration.ProcessModifiers.dd4hep_cff import dd4hep
 from Configuration.Eras.Modifier_run3_egamma_cff import run3_egamma
 from Configuration.Eras.Modifier_run2_egamma_2018_cff import run2_egamma_2018
+from Configuration.Eras.Modifier_run2_HLTconditions_2018_cff import run2_HLTconditions_2018
 from Configuration.Eras.Modifier_run3_RPC_cff import run3_RPC
 
-Run3 = cms.ModifierChain(Run2_2018.copyAndExclude([run2_GEM_2017, ctpps_2018, run2_egamma_2018]),
+Run3 = cms.ModifierChain(Run2_2018.copyAndExclude([run2_GEM_2017, ctpps_2018, run2_egamma_2018, run2_HLTconditions_2018]),
```

Comment:
@missirol @Martin-Grunewald  could you please confirm that this is ok for HLT in Run3?

Reply:
> could you please confirm that this is ok for HLT in Run3?

Yes, it is okay.

To my knowledge, nothing that concerns HLT directly depends on this modifier. It is used in the HLT Offline DQM (FYI: @cms-sw/dqm-l2 (@ckoraka)), e.g.
https://github.com/cms-sw/cmssw/blob/a9b836229a8b2869e3d342b70dd1332e45e0681a/DQMOffline/Trigger/python/TopMonitoring_cff.py

For reference, this change was already discussed in https://github.com/cms-sw/cmssw/pull/39082 and https://github.com/cms-sw/cmssw/pull/39084 (neither of which was merged, despite what the GitHub labels say).

Reply:
Thanks, looking at the discussion in https://github.com/cms-sw/cmssw/pull/39084 it looks like people actually wanted this change to be included then; not clear why it was reverted in https://github.com/cms-sw/cmssw/pull/39119...